install_script.sh.template

```diff
@@ -1122,6 +1127,10 @@ EOF
       formatted_host_tags="['""$( echo "$host_tags" | sed "s/,/','/g" )""']"  # format `env:prod,foo:bar` to yaml-compliant `['env:prod','foo:bar']`
       $sudo_cmd sh -c "sed -i \"s/^# tags:.*/tags: ""$formatted_host_tags""/\" $config_file"
   fi
+  if [ "$dd_env" ]; then
+    printf "\033[34m\n* Adding your DD_ENV to the $nice_flavor configuration: $config_file\n\033[0m\n"
+    $sudo_cmd sh -c "sed -i 's/# env:.*/env: $dd_env/' $config_file"
```

Comment:
There are multiple occurrences of `# env:` in the example config file ([here](https://github.com/DataDog/datadog-agent/blob/52538a086c55208de8031da4a9a0e85f1b3f5705/pkg/config/config_template.yaml#L154) and [here](https://github.com/DataDog/datadog-agent/blob/52538a086c55208de8031da4a9a0e85f1b3f5705/pkg/config/config_template.yaml#L1223)). The logic here should only modify the top-level `env` setting (like what's done for the `tags` config option).

Reply:
Indeed, thanks for noticing! This should now be fixed and accounted for in the tests